packages/paste-design-tokens/tokens/aliases/sizing.yml

```diff
@@ -35,3 +35,5 @@ aliases:
   size-100: "64rem" #1024px
   size-110: "70.5rem" #1128px
   size-120: "77rem" #1264px
+
+  size-sidebar-10: "13.75rem" #220px
```

Comment:
I don't mind the concept of a sharable sidebar width. Especially if we're in the business of making sure our navs are cohesive across products.

But are we really going to have more than one sidebar size? You might for the states, expanded and collapsed, but I'm not sure they they be on a 10 step scale. They're just one off use case tokens

Reply:
Good point. Changed to not include the -10.